mlos_bench/mlos_bench/service/local/__init__.py

```diff
@@ -6,9 +6,11 @@
 Local scheduler side Services for mlos_bench.
 """
 
+from mlos_bench.service.types.local_exec_type import SupportsLocalExec
 from mlos_bench.service.local.local_exec import LocalExecService
 
 
 __all__ = [
+    'SupportsLocalExec',
```

Comment:
Left over from a previous organization I'd tried instead of `mlos_bench.service.types`.
However, it might be worthwhile adding these as a convenience in some of the submodules.
What do you think?